packages/api/core/src/api/start.ts

```diff
@@ -27,6 +27,7 @@ export default async ({
   inspectBrk = false,
 }: StartOptions): Promise<ElectronProcess> => {
   asyncOra.interactive = interactive;
+  asyncOra.keepStdinFlowing = true;
```

Comment:
suggestion: add a brief comment about why this was added and link to the GitHub issue?

Reply:
Nah, just add a link to the issue here. I guess LSP maybe will show the definition docs?